Gemfile

```diff
@@ -1,3 +1,7 @@
 source "https://rubygems.org"
 
 gem 'uswds-jekyll', :git => 'https://github.com/18F/uswds-jekyll.git'
+
+group :jekyll_plugins do
+  gem 'jekyll_pages_api_search'
```

Comment:
should this be removed also (and in the `Gemfile.lock`)?

Reply:
The `Jekyll_pages_api_search` is part of the USWDS-Jekyll theme and removing the gem from the `gemfile` and `gemfile.lock` throws errors. So maybe we can leave it in for now and remove once we remove `Jekyll_pages_api_search` from USWDS-Jekyll.

Reply:
oof, this is bringing back bad memories :P

Reply:
I've got an issue to incorporate search.gov as a search option https://github.com/18F/uswds-jekyll/issues/62

Reply:
It's fine to do that in another PR.